test/config-luatest/credentials_applier_test.lua

```diff
@@ -478,3 +478,54 @@ g.test_sync_privileges = function(g)
 
     child:close()
 end
+
+g.test_set_password = function(g)
+
+    local auth_types = {
+        'chap-sha1',
```

Comment:
I don't insist, but it can be parametrized by luatest facilities.

Reply:
This is outside of scope for this patchset, main discussion [here](https://github.com/tarantool/tarantool/pull/8968#discussion_r1300295265).